Makefile

```diff
@@ -88,6 +88,10 @@ packaging/debian/changelog: packaging/debian/changelog.in
 
 $(WEBPACK_TEST): $(COCKPIT_REPO_STAMP) $(shell find src/ -type f) package.json webpack.config.js
 	$(MAKE) package-lock.json && NODE_ENV=$(NODE_ENV) node_modules/.bin/webpack
+	# In development mode no LICENSE.txt.gz is generated by terser, so we explictly create it as it is required for building rpm's
```

Comment:
More precisely, terser doesn't run at all for devel mode, as it's part of the `optimization:` webpack config.